Examples/Rendering/PBR/index.js

```diff
@@ -98,12 +116,18 @@ reader.setUrl(`${__BASE_PATH__}/data/pbr/helmet.obj`).then(() => {
   const metallicTex = createTexture(`${__BASE_PATH__}/data/pbr/metallic.jpg`);
   const emissionTex = createTexture(`${__BASE_PATH__}/data/pbr/emission.jpg`);
   const normalTex = createTexture(`${__BASE_PATH__}/data/pbr/normal.jpg`);
-
+  const environmentTex = createTextureWithMipmap(
+    `${__BASE_PATH__}/data/pbr/kiara_dawn_4k.png`,
```

Comment:
Because of the fact that this is being used as the background for the scene (and normally only about 1/10th of it is being shown due to field of view), it's very important that it stays as being high quality. Would it be better to store the image somewhere else and then download it when the example starts?

Reply:
Another option would be to try keeping the resolution but increasing the compression? That might work fine.

Reply:
In that case, I would add it to https://github.com/Kitware/vtk-js-datasets and have the example download the image from there.

Reply:
An 80 quality jpg is 1.1 MB and looks really good FWIW

Reply:
Ive updated it to be a pre-tonemapped compressed jpeg, now with a size of 1.6MB